### ISSUE TITLE
Users without photo are shown with broken image in the notification emails

### ISSUE THREAD

**Issue by maxceem** [appirio-tech/connect-app#3706]
This issue is reproducible on PROD.

User doesn't have a photo:

![image](https://user-images.githubusercontent.com/146016/74229149-c479af00-4cfc-11ea-95d5-a58b502eff95.png)

So when such a user create a post and I get a notification, the image is shown as broken:

![image](https://user-images.githubusercontent.com/146016/74229107-ae6bee80-4cfc-11ea-96cc-4e30b9aa2cc4.png)

**Comment by vikasrohit** [comment]
Yes, this is a long time pending issue, I guess. We can not dynamically render Avatar in emails. May be we can link to a default user icon in such case?
